src/renderers/native/NativeMethodsMixin.js

```diff
@@ -150,9 +150,9 @@ var NativeMethodsMixin = {
     );
 
     UIManager.updateView(
-      findNodeHandle(this),
+      (findNodeHandle(this) : any),
       this.viewConfig.uiViewClassName,
-      updatePayload
+      (updatePayload : any)
```

Comment:
These casts smell funny to me. .create() returns ?Object, while updateView returns Object. Can you figure out if one of them is wrong or if we should call .updateView only conditionally? Same with createView elsewhere.